backend/service/authn/authn.go

```diff
@@ -262,12 +271,22 @@ func NewProvider(config *authnv1.Config) (Provider, error) {
 		return nil, err
 	}
 
-	return &OIDCProvider{
+	p := &OIDCProvider{
 		provider:            provider,
 		verifier:            verifier,
 		oauth2:              oc,
 		httpClient:          httpClient,
 		sessionSecret:       config.SessionSecret,
 		claimsFromOIDCToken: DefaultClaimsFromOIDCToken,
-	}, nil
+	}
+
+	if config.Storage != nil {
+		c, err := newCryptographer(config.Storage.EncryptionPassphrase)
```

Comment:
Would it be also beneficial to also support environment variable parsing for`config.Storage.EncryptionPassphrase`?

Reply:
all configuration is already possible via environment variable, no additional support needed

Reply:
Ah yes I dont know what I was thinking.